src/Microsoft.DotNet.Build.Tasks/Targets/resources.targets

```diff
@@ -27,12 +27,21 @@
         ResxFilePath="$(StringResourcesPath)" 
         IntermediateFilePath="$(IntermediateResOutputFileFullPath)" 
         OutputSourceFilePath="$(ResourceSRPath)" 
-        AssemblyName="$(AssemblyName)" >
+        AssemblyName="$(AssemblyName)" 
+        ContinueOnError="ErrorAndStop" >
+        
+        <Output PropertyName="GenerateResourcesCodeOutputMessage" TaskParameter="Message" />
```

Comment:
We can walk through it on Monday but from my local testing it works as I expect and you shouldn't need anything but the Log.LogError.